modules/xmpp/ChatRoom.js

```diff
@@ -467,6 +470,12 @@ export default class ChatRoom extends Listenable {
             case 'jibri-recording-status':
                 jibri = node;
                 break;
+            case 'jibri-sip-status':
```

Comment:
Don't we have a way to register presence node listeners ? It feels like the gateway should be doing that.